### PR TITLE
docs(parrot-middleware): added examples + explanations

### LINKED ISSUE #15: parrot-middleware: Enhance documentation
- [x] Add new branding
- [ ] Document API endpoints. These should include usage examples that illustrate the request and response bodies

### PR DESCRIPTION
**Pull request by amexjake**
This fixes #15

### CODE REVIEW

File: packages/parrot-middleware/README.md [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 parrot-middleware is an implementation of Parrot that can be used as an [Express](http://expressjs.com/) middleware.
 
+Given a collection of Parrot scenarios which each have a list of request and response pairs, the parrot middleware will intercept the `request`s that match the paths in the active scenario, returning the corresponding `response`, in addition to providing the three additional endpoints detailed below for interacting with your scenarios.
```
Comment (anescobar1991) [inline packages/parrot-middleware/README.md]: I think the last bit of the sentence (starting with `in addition to...` is confusing, I am not sure what it is trying to say.

File: packages/parrot-middleware/README.md [hunk 2]
```diff
@@ -16,16 +19,141 @@ app.use(parrot(scenarios));
...
+    },
+  ],
+};
+
+export default scenarios;
+```
+
 Once the server is running, the following endpoints are exposed to interact with your scenarios:
 
+### POST `/parrot/scenario`
```
Comment (anescobar1991) [inline packages/parrot-middleware/README.md]: Can you add a line somewhere that explains how most likely one would want to use the chrome plugin that wraps around this API to switch scenarios?

**Comment by anescobar1991** [comment]
Thanks @amexjake!

File: packages/parrot-middleware/README.md (hunk 1 above)
Comment (amexjake) [reply packages/parrot-middleware/README.md]: Revised this, let me know if you think this is better...

File: packages/parrot-middleware/README.md (hunk 2 above)
Comment (amexjake) [reply packages/parrot-middleware/README.md]: Added a note at the bottom of the readme :D

File: packages/parrot-middleware/README.md [hunk 3]
```diff
@@ -13,19 +65,109 @@ const app = express();
...
+        },
+      },
+    ],
+  },
+  {
+    name: 'has more ships',
+    mocks: [
+      {
+        request: {
+          path: '/ship_log',
```
Comment (jsmartfo) [inline packages/parrot-middleware/README.md]: nitpick - can we make this a little more restful: 

```
/logs/ships/{id}
...
/logs/ships
```

**Comment by jgolden17** [comment]
@amexjake Your branch needs to be rebased on mster before we can merge

File: packages/parrot-middleware/README.md [hunk 4]
```diff
@@ -13,19 +65,109 @@ const app = express();
 
 app.use(parrot(scenarios));
 
-app.listen(3000);
+app.listen(3001);
 ```
 
 Once the server is running, the following endpoints are exposed to interact with your scenarios:
```
[the remark this answers is not in the document]
Comment (amexjake) [reply packages/parrot-middleware/README.md]: Good catch! Looks like I missed this line when I was doing some of the re-writes...I'll fix this.

**Pull request merged**